docs/aws/audit/cloudtrailmonitoring/rules/cloudtrail_publicly_accessible.mdx

```diff
@@ -18,6 +18,32 @@ Security
 
 CIS, PCIDSS, GDPR, APRA, MAS, NIST
 
+### Remediation
+
+How to remove public access to CloudTrail logging bucket?
+
+#### Using AWS Console
+
+1. Log in to the AWS Management Console using your AWS account credentials.
+2. Navigate to the Amazon S3 service by searching for "S3" in the AWS services search bar and selecting "S3" from the results.
+3. In the S3 management console, you will see a list of your S3 buckets.
+4. Identify the bucket(s) that are used for CloudTrail logging and are currently publicly accessible.
```

Comment:
to add directions on how to find these in Cloudanix, let's add one more sentence at the end. `In the Cloudanix Console, navigate to "Misconfig" page and look for Affected Assets for "CloudTrail Logging Bucket Should Use MFA Delete Feature" Policy`.